controllers/chat/initChatFromPost.js

```diff
@@ -114,9 +114,12 @@ const initChatFromPost = async (req, res) => {
     try {
       if (!newChannel?.data?.name) {
         let channelType = CHANNEL_TYPE.CHAT;
-        let anonDetailInfo = {};
-        if (userModel.is_anonymous || targetUserModel.is_anonymous) {
+        if (userModel.is_anonymous) {
           channelType = CHANNEL_TYPE.ANONYMOUS;
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in logic for setting the `channelType` to `CHANNEL_TYPE.ANONYMOUS` only when `userModel.is_anonymous` is true seems fine. However, it's important to ensure that this doesn't break any existing functionality where `channelType` was previously set to `ANONYMOUS` if either `userModel` or `targetUserModel` were anonymous.



<!-- This is an auto-generated comment by OSS CodeRabbit -->